tests/test_client.py

```diff
@@ -1201,6 +1201,21 @@ def test_exchange_success(self):
         self.assertEqual('dummy_revoke_uri', credentials.revoke_uri)
         self.assertEqual(set(['foo']), credentials.scopes)
 
+    def test_exchange_success_binary_code(self):
+        payload = (b'{'
+                   b'  "access_token":"SlAV32hkKG",'
```

Comment:
Uhh...

... after some searching: [`ServiceAccountCredentialsTest.test_from_json_keyfile_name_factory`](https://github.com/google/oauth2client/blob/b605997beaa6361ab0811ffe62dfaf8de0eed03e/tests/test_service_account.py#L110). Although even there I'd object to the literal strings "foo" and "bar" each appearing more than once in the test data set.

But do you see what I mean about method-local constants, so that the literal values used as the test inputs are only written once?